.github/workflows/build.yml

```diff
@@ -5,10 +5,12 @@ on:
     branches:
       - main
       - release-*
+      - feature/*
   push:
     branches:
       - main
       - release-*
+      - feature/*
```

Comment:
I suppose you need to update all jobs with extra step similar like below? otherwise, it will do nothing when it's from feature branch for some builds.
```
    - name: Build Antrea amd64 Docker image without pushing to registry
      if: ${{ github.repository != 'antrea-io/antrea' || github.event_name != 'push' || github.ref != 'refs/heads/main' }}
```

Reply:
I agree with @XinShuYang. I don't think we need to change anything. We are treating `feature/*` branches the same way as `release/*` branches when it comes to running tests.

Reply:
ah, yes, my bad, I thought some of the steps are build binary only instead of build image. ignore me.